Makefile

```diff
@@ -35,11 +35,19 @@ circle_ci_setup:
 	curl -X POST https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/follow?circle-token=${CIRCLECI_API_KEY}
 
 github_actions_setup:
-	sh scripts/gha-setup.sh
+	sh scripts/gha-setup.sh setup
 
 summary:
 	@echo "zero-deployable-backend:"
 	@echo "- Repository URL: ${REPOSITORY}"
 	@echo "- Deployment Pipeline URL: https://app.circleci.com/pipelines/github/${GITHUB_ORG}/${GITHUB_REPO}"
 	@echo $(shell echo ${ENVIRONMENT} | grep prod > /dev/null && echo "- Production API: ${productionBackendSubdomain}${productionHostRoot}")
 	@echo $(shell echo ${ENVIRONMENT} | grep stage > /dev/null && echo "- Staging API: ${stagingBackendSubdomain}${stagingHostRoot}")
+
+REQUIRED_BINS_GHA := gh
+check:
+ifeq ($(CIVendor), github-actions)
+	$(foreach bin, $(REQUIRED_BINS_GHA),\
+			$(if $(shell command -v $(bin) 2> /dev/null),$(info Found `$(bin)`),$(error Please install `$(bin)`)))
```

Comment:
In the application do we have any help text around this? Telling them we are looking for binaries on their system that are required for modules, maybe telling them how to install stuff? That's a little harder as for example some things might use brew but others have to be downloaded, etc.
Any thoughts about how to provide this in a friendly way?

Reply:
with this it will error with a message like this
![image](https://user-images.githubusercontent.com/1787590/113756269-2fdfc680-96df-11eb-83f3-418bacd33e88.png)

Reply:
This looks a bit weird, we must be able to provide something better. I wonder if we should just call a script or something rather than make, or suppress some of that additional output from make.

Reply:
yeah we could have a generic script thats used for checking binaries
since it doesn't get templated into the user's repo we can have scripts like that 👍

Reply:
After your change it's still being run by the makefile, does that mean we'll still get the weird output from make? (`Makefile:50 ...`, `Stop.1, etc.)

Reply:
that might need to be parsed out from the Zero side, the output looks different when i run the makefile itself

Reply:
not sure if there's a way to suppress the makefile outputs

Reply:
It does have a `--quiet` flag but I think that just forces it not to echo commands, and doesn't change its other output.

Reply:
Maybe we should just have a script that zero calls directly rather than using the makefile?
We could potentially even have that as part of the module.yaml.. 
```
applyScript: make apply
checkScript: scripts/check.sh
```
or something.
Sorry, throwing some curve balls in here.

Reply:
hmm that could be useful in other scenarios too, becomes more like scripts in the package.json in nodejs